src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public enum Color {
+        BLACK,
+        WHITE,
+        RED,
+        BLUE,
+        GREEN,
+        YELLOW
+    }
+
+    public static Color getRandomColor() {
+        Random random = new Random();
+        return Color.values()[random.nextInt(Color.values().length)];
```

Comment:
Let's create an `randomIndex` as a variable.
```suggestion
        int randomIndex = random.nextInt(Color.values().length);
        return Colors.values()[randomIndex];
```